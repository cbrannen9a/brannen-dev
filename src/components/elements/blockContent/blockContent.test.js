import React from 'react';
import renderer from 'react-test-renderer';

import BlockContent from './blockContent';

const blocks = [{
	style: 'normal', _key: 'block1', _type: 'block', markDefs: [],
	children: [
		{
			marks: [],
			text: 'test text',
			_key: 'mock1',
			_type: 'span'
		},
		{
			marks: ['strong'],
			text: 'test bold',
			_key: 'mock2',
			_type: 'span'
		},
		{
			marks: ['em'],
			text: 'test italic',
			_key: 'mock3',
			_type: 'span'
		}]
}];

describe('BlockContent', () => {
	it('renders correctly', () => {
		const tree = renderer
			.create(<BlockContent blocks={blocks} />)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});