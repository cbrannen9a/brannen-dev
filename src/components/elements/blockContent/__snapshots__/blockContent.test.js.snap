// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlockContent renders correctly 1`] = `
<p>
  test text
  <strong>
    test bold
  </strong>
  <em>
    test italic
  </em>
</p>
`;
